architecture/additional_concepts/other_api_objects.adoc

```diff
@@ -249,18 +249,14 @@ this token, if any.
 
 === Identity
 When a user logs into OpenShift, they do so using a configured
-link:../../admin_guide/configuring_authentication.html#identity-providers[identity
-provider]. This determines the user's identity, and provides that information to
-OpenShift.
+link:../../admin_guide/configuring_authentication.html#identity-providers[identity provider].
+This determines the user's identity, and provides that information to OpenShift.
 
 OpenShift then looks for a `*UserIdentityMapping*` for that `*Identity*`:
 
-- If the `*Identity*` already exists, but is not mapped to a `*User*`, login
-fails.
-- If the `*Identity*` already exists, and is mapped to a `*User*`, the user is
-given an `*OAuthAccessToken*` for the mapped `*User*`.
-- If the `*Identity*` does not exist, an `*Identity*`, `*User*`, and
-`*UserIdentityMapping*` are created, and the user is given an
+* If the `*Identity*` already exists, but is not mapped to a `*User*`, then login fails.
+* If the `*Identity*` already exists, and is mapped to a `*User*`, then OpenShift gives the user an `*OAuthAccessToken*` for the mapped `*User*`.
+* If the `*Identity*` does not exist, then OpenShift creates an `*Identity*`, `*User*`, and `*UserIdentityMapping*`. OpenShift also gives the user an
 `*OAuthAccessToken*` for the mapped `*User*`.
```

Comment:
FWIW, I've been sticking with `-`s for lists, as I've seen `*`s get funky in certain situations with other `*`s on the same line (not that it seems to affect this particular example).